src/pages/homepage/homepage.component.jsx

```diff
@@ -34,18 +34,18 @@ class HomePage extends React.Component {
 
   render() {
     const { pokemon, searchField } = this.state;
-    const filteredPokemon = pokemon.filter((monster) => (
+    const filteredPokemon = pokemon.filter(monster => (
```

Comment:
nit: no parens needed for 1 param

Reply:
Is it bad practice to leave the parenthesis in?